flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/TestFlinkCatalogTable.java

```diff
@@ -188,6 +188,37 @@ public void testCreateTableLike() throws TableNotExistException {
         .isEqualTo(TableSchema.builder().field("id", DataTypes.BIGINT()).build());
   }
 
+  @TestTemplate
+  public void testCreateTableLikeInDifferentCatalog() throws TableNotExistException {
```

Comment:
isn't this the same test as the method above it?

Reply:
This is to create table using LIKE in different Iceberg Catalog, where as previous one was create dynamic table in FlinkCatalog.

Reply:
I don't see a different Iceberg catalog. this test seems identical to the one above.

Reply:
copy/paste error. Updated now. Thanks for catching it.